packages/README.md

```diff
@@ -0,0 +1,46 @@
+# Packages
+
+This directory contains two types of packages. Management Packages and Standalone Packages
+
+## Management Packages
+
+This directory contains some carvel packages for management components. Any package that performs the role of managing and operating a workload cluster is a Management package. This management packages are purpose-built to be installed on the management cluster. More details about packages can be found [here](../docs/packages/definitions.md)
+
+This management packages are associated with each in terms of [Meta Package](../docs/packages/definitions.md#meta-package). A Meta Package contains PackageInstall CRs, Secrets and ServiceAccounts, RBAC rules that are needed to install other packages and when the meta package is installed, it installs the other packages.
```

Comment:
"This management packages" -> "These management packages"

There seems to be a word missing in "This management packages are associated with each in terms".  With "each" what?